packages/reference/src/common/ReferenceEditor.tsx

```diff
@@ -16,6 +17,7 @@ export interface ReferenceEditorProps {
   sdk: FieldExtensionSDK;
   viewType: ViewType;
   renderCustomCard?: (props: CustomEntryCardProps) => React.ReactElement | false;
+  renderCustomActions?: (props: CustomActionProps) => React.ReactElement;
```

Comment:
Maybe it's worth to support the same `false` logic for consistency with `renderCustomCard` property?

Reply:
That is what I had in the beginning. After asking @DanweDE about the rationale behind the support for false for the card I didn't feel like we actually need it.